src/main/java/org/jenkinsci/plugins/ghprb/GhprbPullRequest.java

```diff
@@ -78,6 +78,9 @@
 
     private final int id;
     private Date updated; // Needed to track when the PR was updated
+    // Local representation of the number of comments that we have
+    // on the PR.
+    private transient int commentCount;
```

Comment:
Note to other maintainers.  In Jenkins plugins, `transient` means the `commentCount` value _will not_ be serialized to disk as XML (i.e. it will not be saved to XML configuration).  This value will not persist across Jenkins master restarts.